Tasks/DownloadGitHubNpmPackageV1/task.json

```diff
@@ -10,7 +10,7 @@
     "version": {
         "Major": 1,
         "Minor": 206,
-        "Patch": 0
+        "Patch": 1
```

Comment:
I wonder if there's a bug in the bump-task script - this should be 208.0 as well